fees-rewards.md

```diff
@@ -0,0 +1,77 @@
+# Fees and Rewards
+
+Obscuro sets out a simple structure for fees and rewards. End users pay fees based on their usage which are placed into a pool from which rewards for nodes are paid out.
+
+The structure delivers a steady and predictable income for node operators over the long term and a predictable fee for users proportional to the actual costs of running the network and does not rely on bidding wars or vary according to activity.
+
+The structure also doesn&#39;t punish Aggregators that miss out on a publishing opportunity with slashing. Instead, the loss of reward should be incentive enough.
+
+## Cost of L2 nodes
+
+Operators run the network and incur operational costs by running and maintaining nodes.
+
+An &quot;expected monthly node operational costs&quot; variable is set through the governance protocol, which is designed to reflect how much a node operator would pay for hardware, hosting etc, in a month. This is analogous to the proof of work difficulty found in other blockchains. This effects demand for running a node.
+
+Aggregator nodes incur cost from submitting transactions to L1, through the following:
+
+- Storing rollup data on Ethereum.
+- The overhead in participating in publishing rollups outside of the storage cost, e.g. signing transactions.
```

Comment:
- the overhead of processing the Ethereum transaction that contains the rollup.

Reply:
this doesn't look resolved

Reply:
Try refreshing, it's resolved